.github/workflows/chronograf-cypress-tests.yaml

```diff
@@ -0,0 +1,91 @@
+name: "chronograf/cypress"
+
+on:
+  pull_request:
+  schedule:
+    - cron: '30 5 * * 1-5'
+
+jobs:
+  e2e:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: '1.16.4'
+
+      - name: Setup Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.6.3
+
+      - name: Create kind cluster
+        uses: helm/kind-action@v1.2.0
+        with:
+          cluster_name: "chronograf-testing"
+          config: .github/workflows/resources/kind-config.yaml
+
+      - name: Create Enterprise Test Resources
+        run: |
+          helm repo add jetstack https://charts.jetstack.io
+          helm repo add influxdata https://helm.influxdata.com/
+          helm repo update
+          helm upgrade --wait --install \
+              cert-manager jetstack/cert-manager \
+              --namespace cert-manager \
+              --create-namespace \
+              --version v1.5.4 \
+              --set prometheus.enabled=false \
+              --set webhook.timeoutSeconds=30 \
+              --set installCRDs=true
+          kubectl apply -f .github/workflows/resources/test-resources.yaml
+          kubectl create secret generic influxdb-license --from-literal=INFLUXDB_ENTERPRISE_LICENSE_KEY=${INFLUXDB_ENTERPRISE_LICENSE_KEY}
+          helm upgrade --install influxdb influxdata/influxdb-enterprise --namespace default --set-string envFromSecret=influxdb-license --set-string data.service.type=NodePort
+          kubectl patch svc influxdb-influxdb-enterprise-data --type=json -p '[{"op":"replace","path":"/spec/ports/0/nodePort","value":30086}]'
+        env:
+          INFLUXDB_ENTERPRISE_LICENSE_KEY: "${{ secrets.INFLUXDB_ENTERPRISE_LICENSE_KEY }}"
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14.15.0'
+      - run: npm install --global yarn
```

Comment:
Please follow https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#caching-packages-dependencies ... yarn should be already installed

Reply:
https://github.com/actions/setup-node/issues/182 - actions/setup-node does not install yarn - yarn should be pre-installed on official github runner images which we already use, but I'm not able to test it locally via act tool (https://github.com/nektos/act). `[chronograf/cypress/e2e]   💬  ::debug::Caching tool node 14.15.0 x64
[chronograf/cypress/e2e]   💬  ::debug::source dir: /tmp/06314f42-d06c-40f4-91e2-2672679cd5aa
[chronograf/cypress/e2e]   💬  ::debug::destination /opt/hostedtoolcache/node/14.15.0/x64
[chronograf/cypress/e2e]   💬  ::debug::finished caching tool
| Done
[chronograf/cypress/e2e]   ❗  ::error::Unable to locate executable file: yarn. Please verify either the file path exists or the file can be found within a directory specified by the PATH environment variable. Also check the file mode to verify the file is executable.
[chronograf/cypress/e2e]   ❌  Failure - actions/setup-node@v2`

Reply:
Ok, undestood ... `if [ -z $(which yarn) ]; then npm install --global yarn ; fi` might be a solution